modules/installation-configuration-parameters.adoc

```diff
@@ -766,11 +766,16 @@ host must trust the certificate.
 |`platform.aws.userTags`
 |A map of keys and values that the installation program adds as tags to all resources that it creates.
 |Any valid YAML map, such as key value pairs in the `<key>: <value>` format. For more information about AWS tags, see link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html[Tagging Your Amazon EC2 Resources] in the AWS documentation.
+[NOTE]
+====
+You can add upto 25 user defined tags during installation. The ratio is 25:25 that is user defined tags: reserved {product-title} tags.  
```

Comment:
It is not a ratio. It should be "Remaining tags are reserved for Open Shift"

Reply:
I have specified the tags , ptal